homeassistant/components/device_tracker/bmw_connected_drive.py

```diff
@@ -0,0 +1,51 @@
+"""Device tracker for BMW Connected Drive vehicles.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/device_tracker.bmw_connected_drive/
+"""
+import logging
+
+from homeassistant.components.bmw_connected_drive import DOMAIN \
+    as BMW_DOMAIN
+from homeassistant.util import slugify
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['bmw_connected_drive']
+
+
+def setup_scanner(hass, config, async_see, discovery_info=None):
+    """Set up the BMW tracker."""
+    entities = hass.data[BMW_DOMAIN]
+    _LOGGER.debug('Found BMW accounts: %s',
+                  ', '.join([v.name for v in entities]))
+    for entity in entities:
```

Comment:
These are not entities. Please rename the variable. Eg use cars.

Reply:
done